WebDriverAgentLib/Utilities/FBAccessibilityTraits.m

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <XCTest/XCTest.h>
+
+NSArray<NSString *> *FBAccessibilityTraitsToStringsArray(unsigned long long traits) {
+    NSMutableArray<NSString *> *traitStringsArray;
+    NSNumber *key;
+    
+    static NSDictionary<NSNumber *, NSString *> *traitsMapping;
+    static dispatch_once_t onceToken;
+
+    dispatch_once(&onceToken, ^{
+        NSMutableDictionary<NSNumber *, NSString *> *mapping = [@{
+            @(UIAccessibilityTraitNone): @"None",
+            @(UIAccessibilityTraitButton): @"Button",
+            @(UIAccessibilityTraitLink): @"Link",
+            @(UIAccessibilityTraitHeader): @"Header",
+            @(UIAccessibilityTraitSearchField): @"SearchField",
+            @(UIAccessibilityTraitImage): @"Image",
+            @(UIAccessibilityTraitSelected): @"Selected",
+            @(UIAccessibilityTraitPlaysSound): @"PlaysSound",
+            @(UIAccessibilityTraitKeyboardKey): @"KeyboardKey",
+            @(UIAccessibilityTraitStaticText): @"StaticText",
+            @(UIAccessibilityTraitSummaryElement): @"SummaryElement",
+            @(UIAccessibilityTraitNotEnabled): @"NotEnabled",
+            @(UIAccessibilityTraitUpdatesFrequently): @"UpdatesFrequently",
+            @(UIAccessibilityTraitStartsMediaSession): @"StartsMediaSession",
+            @(UIAccessibilityTraitAdjustable): @"Adjustable",
+            @(UIAccessibilityTraitAllowsDirectInteraction): @"AllowsDirectInteraction",
+            @(UIAccessibilityTraitCausesPageTurn): @"CausesPageTurn",
+            @(UIAccessibilityTraitTabBar): @"TabBar"
+        } mutableCopy];
+        
+        #if __clang_major__ >= 15 || (__clang_major__ >= 14 && __clang_minor__ >= 0 && __clang_patchlevel__ >= 3)
```

Comment:
I'm not sure what actual Xcode version (clang) can build them, but I assume at least:

```suggestion
        #if __clang_major__ >= 16
```

Reply:
Let me apply this.